### PR TITLE
Fix/ui modal qunit test

### PR DESCRIPTION
**Pull request by zagovorichev**
https://oat-sa.atlassian.net/browse/TAO-5161

### CODE REVIEW

File: views/js/test/ui/modal/test.js [hunk 1]
```diff
@@ -189,9 +188,8 @@ define([
         $container.append($scrolledContainer);
         $scrolledContainer.scrollTop(500);
 
-        $modal.on('opened.modal', function() {
-            assert.ok(true, "The modal is visible");
-            assert.equal($modal.css('top'), '540px', 'Modal window opened in the scrolled position');
+        $modal.on('animate.modal', function(op, to) {
+            assert.equal(to.top, '540px', 'Modal window opened in the scrolled position');
```
Comment (ssipasseuth) [inline views/js/test/ui/modal/test.js]: I am afraid that testing now the to.top value is not as good as testing the  $modal.css('top') you have before. Indeed $modal.css('top') tests really what the users sees whereas testing the value to.top does not offer the warranty that the modal is properly positioned. Would you agree ?

File: views/js/ui/modal.js [hunk 2]
```diff
@@ -240,6 +242,7 @@ define(['jquery', 'core/pluginifier', 'core/dataattrhandler'], function ($, Plug
 
                 $element.show();
 
+                $element.trigger('animate.' + pluginName, to);
                 if (options.animate && $element.is(':visible')) {
```
Comment (ssipasseuth) [inline views/js/ui/modal.js]: so I guess that the actually issue is due to the fact that the callback onOpen() in `$element.animate(to, options.animate, onOpen);` is called. The real fix, isn't to understand why ? maybe something in the doc of $.animate would gives some hint ?

File: views/js/test/ui/modal/test.js (hunk 1 above)
Comment (zagovorichev) [reply views/js/test/ui/modal/test.js]: I think that the cause of this is PhantomJs, I don't know why top always auto

Comment (zagovorichev) [reply views/js/test/ui/modal/test.js]: I found the reason of this (we need position absolute for the element to move it)

Comment (zagovorichev) [reply views/js/test/ui/modal/test.js]: I'm going to prepare new PR, this one will be closed

Comment (zagovorichev) [reply views/js/test/ui/modal/test.js]: https://github.com/oat-sa/tao-core/pull/1501

Comment (ssipasseuth) [reply views/js/test/ui/modal/test.js]: Ok thanks !

**Pull request closed without merging**